domain-models-runtime/src/test/java/org/folio/rest/persist/PgUtilIT.java

```diff
@@ -1405,15 +1404,7 @@ private UserdataCollection searchForDataUnoptimizedNoClass(String cql, int offse
     PgUtil.get(
         "users", User.class, Object.class, cql, offset, limit, okapiHeaders,
         vertx.getOrCreateContext(), ResponseImpl.class, testContext.asyncAssertSuccess(response -> {
-          if (response.getStatus() != 500) {
-            testContext.fail("Expected status 500, got "
-                + response.getStatus() + " " + response.getStatusInfo().getReasonPhrase());
-            async.complete();
-            return;
-          }
-          UserdataCollection c = (UserdataCollection) response.getEntity();
-          userdataCollection.setTotalRecords(c.getTotalRecords());
-          userdataCollection.setUsers(c.getUsers());
```

Comment:
If userdataCollection is not filled with the results the method should return void.